### DISCUSSION TITLE
BLOCK REFERENCES LIKE IN OBSIDIAN & LOGSEQ

### DISCUSSION THREAD

**Discussion by m1e2d3** [TriliumNext/Trilium#2725] [Ideas] [answered]
### Hello sir, 
Can you also add the feature of Block refrences in trillium like in the **logseq** or **obsidian** e.g. for a long note having multiple headings. 
So far it's the best note taking app i have come acrossed till date.

**Comment by XXXJetfireXXX** [comment, marked as answer]
Like zadam said in #1021:
> while this is a valid feature request, it's quite a bit more complex than one could guess from outside. So don't get your hopes up, it's probably not coming in near or medium term future.

As of now, you can split the large note into smaller parts and reference the needed one, or you can use "Cut and paste selection to subnote" and include it in the parent note to keep your text more consistent.
![image](https://user-images.githubusercontent.com/43646178/158110928-b5e2f13d-3dee-4ac4-ba55-7233515be3b8.png)
![image](https://user-images.githubusercontent.com/43646178/158110950-78fc089f-df7c-411c-9741-e332475260ec.png)

After that you can reference it with a link as well.
![image](https://user-images.githubusercontent.com/43646178/158111385-2b0e7fdc-c35a-4ed8-86de-003146a83572.png)

**Reply by m1e2d3** [reply]
Thanks for help GOOD IDEA

**Reply by XXXJetfireXXX** [reply]
Glad to help!
